examples/terraform/azure/main.tf

```diff
@@ -132,20 +118,39 @@ resource "azurerm_network_security_group" "sg" {
     source_address_prefix      = "*"
     destination_address_prefix = "*"
   }
+
+  security_rule {
+    name                       = "kubeapi"
+    description                = "Allow inbound kubeAPI"
+    priority                   = 1020
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = "*"
+    destination_port_range     = "6443"
+    source_address_prefix      = "*"
+    destination_address_prefix = "*"
+  }
+
   tags = {
     environment = "kubeone"
     cluster     = var.cluster_name
   }
 }
 
+resource "azurerm_subnet_network_security_group_association" "sg_subnet" {
+  subnet_id                 = azurerm_subnet.subnet.id
+  network_security_group_id = azurerm_network_security_group.sg.id
+}
+
 resource "azurerm_public_ip" "lbip" {
   count = local.loadbalancer_count
 
   name                = "${var.cluster_name}-lbip"
   location            = var.location
   resource_group_name = azurerm_resource_group.rg.name
-  allocation_method   = "Dynamic"
-  sku                 = var.ip_sku
+  allocation_method   = "Static"
```

Comment:
Can we also add a variable for this?

Reply:
Problem is, it doesn't work with Basic anymore. So does it make sense to parametrize it?